src/tike/ptycho/solvers/divided.py

```diff
@@ -21,40 +22,44 @@ def divided(
     least-squares solver for generalized maximum-likelihood ptychography.
     Optics Express. 2018.
 
-    .. seealso:: tike.ptycho.combined
-
     """
-    nearplane = self.diffraction.fwd(psi=psi, scan=scan, probe=probe)
+    farplane = op.fwd(psi=psi, scan=scan, probe=probe)
+
+    farplane, cost = update_phase(op, data, farplane, num_iter=cg_iter)
```

Comment:
cd_iter -> iter?

Reply:
No. cg_iter is the number of conjugate gradient iterations for each subproblem. This is not the same and the number of iterations for the whole problem.